src/sentry/sentry_metrics/querying/data_v2/execution.py

```diff
@@ -249,30 +254,33 @@ def _align_date_range(cls, metrics_query: MetricsQuery) -> MetricsQuery:
 
 @dataclass(frozen=True)
 class QueryResult:
-    series_executable_query: MetricsQuery | None
-    totals_executable_query: MetricsQuery | None
+    series_query: ScheduledQuery | None
+    totals_query: ScheduledQuery | None
```

Comment:
Decided to re-embed again the scheduled query, in order to access fields directly, without having to pass them across the code.